packages/admin/cms-admin/src/form/file/DamPathLazy.tsx

```diff
@@ -0,0 +1,23 @@
+import { gql, useQuery } from "@apollo/client";
+import React from "react";
+
+import { GQLDamPathLazyQuery, GQLDamPathLazyQueryVariables } from "../../graphql.generated";
+
+export const damPathLazyQuery = gql`
```

Comment:
```suggestion
const damPathLazyQuery = gql`
```

No need to export this query